src/windows/new_ride.c

```diff
@@ -290,7 +290,7 @@ static void window_new_ride_populate_list()
 
 	uint8 currentCategory = _window_new_ride_current_tab;
 	ride_list_item *nextListItem = (ride_list_item*)0x00F43523;
-	rct_ride_type **rideEntries = (rct_ride_type**)0x009ACFA4;
+	rct_ride_type **rideEntries = (rct_ride_type**)RCT2_ADDRESS_RIDE_ENTRIES;
```

Comment:
Shouldn't it be using https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/ride/ride.h#L862 ?